### PR TITLE
Add manim

### PR DESCRIPTION
**Pull request by naveen521kk**
replace python-color
update python-manimpango

### CODE REVIEW

File: mingw-w64-manim/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+source=("${_realname}-${pkgver}.tar.gz"::"https://github.com/ManimCommunity/${_realname}/releases/download/v${pkgver}/${_realname}-${pkgver}.tar.gz")
+sha512sums=('87aa8db29a97d44029082b3a6607beb70277904475ad8fe4d1a2df4bcc6e455b68aae3e26bd36f9b3f9663d5219e32a5f84e188bc1089aa0f6b7537260bc0d97')
+
+prepare() {
+  rm -rf python-build-${CARCH} | true
+  cp -r "${_realname}-${pkgver}" "python-build-${CARCH}"
+  cd "python-build-${CARCH}"
+	sed -i "/'progressbar',/d" setup.py
+	sed -i 's/rich>=6.0,<7.0/rich/' setup.py
+  cd ..
```
Comment (Biswa96) [inline mingw-w64-manim/PKGBUILD:37]: `cd ..` is not required. build step will automatically start with `$srcdir`.

**Comment by naveen521kk** [comment]
https://github.com/msys2/MINGW-packages/runs/1710755758?check_suite_focus=true#step:5:29
```
warning: mingw-w64-python-colour: unrecognized license ‘BSD-3-Clause’
```
what should it be?

**Comment by lazka** [comment]
that's fine.. ideally pacman would use spdx for licenses, but they aren't there yet.

**Comment by lazka** [comment]
you need to add the python-colour-science package back

File: mingw-w64-python-colour-science/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,46 @@
+# Contributor: Brien Dieterle <briend@gmail.com>
+
+_realname=colour-science
+_pypkgname=colour
+pkgbase=mingw-w64-python-${_realname}
+pkgname=("${MINGW_PACKAGE_PREFIX}-python-${_realname}")
+conflicts=("${MINGW_PACKAGE_PREFIX}-python-${_pypkgname}")
```
Comment (naveen521kk) [inline mingw-w64-python-colour-science/PKGBUILD:7]: @lazka What to do here? CI is failing?

Comment (lazka) [reply mingw-w64-python-colour-science/PKGBUILD:7]: it's expected with how things are handled in CI right now..

**Pull request merged**